pulsar-broker/src/main/java/org/apache/pulsar/broker/loadbalance/impl/ModularLoadManagerImpl.java

```diff
@@ -730,7 +730,7 @@ private void updateBundleSplitMetrics(Set<String> bundlesToBeSplit) {
     }
 
     private static final Summary selectBrokerForAssignment = Summary.build(
-            "pulsar_broker_load_manager_bundle_assigment", "-")
+            "pulsar_broker_load_manager_bundle_assignment", "-")
```

Comment:
@sijia-w please confirm this change with engineer, thanks.

Reply:
I'm not sure we need to modify this indicator name, because maybe someone use this name somewhere. /cc @merlimat PLTA

Reply:
Hi @sijia-w, confirmed w/ @gaoran10, please do not modify the code files since they may be referenced somewhere. Can you revert this change?